pkg/cvo/sync_worker.go

```diff
@@ -374,15 +375,23 @@ type statusWrapper struct {
 
 func (w *statusWrapper) Report(status SyncWorkerStatus) {
 	p := w.previousStatus
+	var percentComplete float32
+	if status.Total > 0 {
+		percentComplete = float32(status.Done) / float32(status.Total)
```

Comment:
nit: these would need `*100` to become percent; as it stands they should be `fractionComplete` and `previousFractionComplete`.  I think most folks will get the idea, though, so feel free to pull the hold if you want to leave it alone and skip the retests.

/hold

Reply:
Fixed.